src/ApiService/ApiService/QueueNodeHearbeat.cs

```diff
@@ -39,9 +39,8 @@ public class QueueNodeHearbeat {
         if (!r.IsOk) {
             var (status, reason) = r.ErrorV;
             log.Error($"Failed to replace heartbeat info due to [{status}] {reason}");
+        } else {
+            await _events.SendEvent(new EventNodeHeartbeat(node.MachineId, node.ScalesetId, node.PoolName));
```

Comment:
didn't we discuss and decided to have 1 to 1 mapping with python ? 

Which means we still the event (since in Python it is not validated whether table update succeeded or not)

Reply:
You are right i missread the python code. This change is not necessary